core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2073,6 +2097,13 @@ public String toString() {
     public static final String PROXY_WEB_PORT = "alluxio.proxy.web.port";
 
     //
+    // Log server related properties
+    //
+    public static final String LOGSERVER_BASE_LOGS_DIR = "alluxio.logserver.baselogsdir";
```

Comment:
can this be something like `alluxio.logserver.logs.dir `?

Reply:
Done.